meta/duncrow-gmbh/logo.svg

```diff
@@ -0,0 +1,99 @@
+			</g>
+			<g>
+				<path class="st2" d="M246.5,630.4c-2.9,1.3-5.4,3.1-7.6,5.5c-2.2,2.4-3.7,5.5-4.8,9.3c-1,3.8-1.2,8.5-0.5,13.9
+					c0.4,3.3,1.9,6.8,4.5,10.3c2.6,3.6,6.1,6.6,10.6,9.1c4.5,2.5,9.8,4.1,16.1,4.7c6.3,0.6,13.3-0.4,21.1-3l0.3,4.7
+					c-2.1,0.9-4,1.7-5.9,2.6c-1.9,0.8-4,1.6-6.3,2.3c-2.3,0.7-5.1,1.2-8.3,1.7c-3.2,0.4-7.1,0.7-11.7,0.8
+					c-5.3,0.1-10.1-0.8-14.5-2.6c-4.4-1.8-8.2-4.3-11.5-7.6c-3.2-3.2-5.7-7.1-7.5-11.6c-1.8-4.5-2.6-9.4-2.6-14.7
+					c0-6.3,1.2-12,3.7-17.2c2.5-5.2,5.9-9.5,10.4-12.8s9.6-5.7,15.6-7c6-1.3,12.5-1.1,19.5,0.6c5.5,1.8,9.6,4,12.5,6.6
+					c2.8,2.6,4.8,5.1,5.9,7.3c0.2,0.4,0.3,0.6,0.4,0.5c0,0-0.2,0-0.9,0.3c-0.6,0.2-1.7,0.8-3.1,1.8c-1.4,0.9-3.5,2.6-6.2,4.9
+					c-1.4-2.2-2.6-4.2-3.8-5.9c-1.2-1.7-2.5-3-4-4.1c-1.5-1.1-3.2-1.9-5.1-2.3c-2-0.5-4.4-0.7-7.2-0.7
+					C252.4,628.3,249.4,629.1,246.5,630.4z"/>
+				<path class="st2" d="M343.7,617.3c1.2,0,2.3,0.1,3.4,0.1v7.6c-1.2-0.2-2.4-0.3-3.7-0.4c-1.3,0-2.5-0.1-3.7-0.1
+					c-3-0.1-6.1,0.3-9.1,1.1c-3,0.8-5.8,2.2-8.2,4.2c-2.4,2-4.4,4.6-5.9,7.8c-1.5,3.3-2.3,7.4-2.3,12.2v40.6h-16.9v-34.8
+					c0-11.9,3.7-21.3,11.1-28.2c7.4-6.9,18.1-10.3,32.1-10.3C341.5,617.3,342.6,617.3,343.7,617.3z"/>
+				<path class="st2" d="M409.7,620.6c4.5,2.1,8.2,4.8,11.2,8.1c2.9,3.4,5.1,7.3,6.6,11.7s2.3,9,2.5,13.8c-0.2,5-1,9.7-2.4,14.2
+					c-1.4,4.5-3.6,8.3-6.5,11.6c-2.9,3.3-6.7,5.9-11.2,7.8c-4.6,2-10.1,2.9-16.5,2.9c-6.8,0-12.5-1-17.3-2.9c-4.8-2-8.6-4.6-11.7-8
+					c-3-3.4-5.3-7.3-6.8-11.7s-2.3-9-2.5-13.9c0.1-4.8,0.9-9.4,2.3-13.8s3.7-8.3,6.8-11.7c3-3.4,7-6.1,11.7-8.1
+					c4.8-2.1,10.6-3.1,17.3-3.1C399.8,617.5,405.2,618.5,409.7,620.6z M373.5,668.9c1,3.3,2.4,5.9,4.2,7.8c1.8,2,4,3.3,6.5,4.2
+					c2.5,0.8,5.3,1.2,8.4,1.1c3,0.1,5.8-0.2,8.4-1c2.5-0.8,4.7-2.1,6.5-4.1c1.8-2,3.2-4.6,4.2-7.9c1-3.3,1.5-7.4,1.5-12.3v-5.1
+					c0-4.9-0.5-9-1.5-12.3c-1-3.3-2.4-6-4.2-7.9c-1.8-2-4-3.3-6.5-4.1c-2.5-0.8-5.3-1.1-8.4-1c-3-0.1-5.8,0.3-8.4,1.1
+					c-2.5,0.8-4.7,2.2-6.5,4.2c-1.8,2-3.2,4.6-4.2,7.8c-1,3.3-1.5,7.4-1.5,12.2v5.1C372,661.5,372.5,665.6,373.5,668.9z"/>
+				<path class="st2" d="M460.6,668.9c0.8,3.3,2,5.9,3.5,7.8c1.6,2,3.5,3.3,5.9,4.2c2.3,0.8,5,1.2,8.1,1.1c3,0.1,5.8-0.2,8.4-1
+					c2.5-0.8,4.7-2.1,6.5-4.1c1.8-2,3.2-4.6,4.2-7.9c1-3.3,1.5-7.4,1.5-12.3c0-0.4,0-1-0.1-1.8c-0.1-0.8-0.1-2.6-0.1-5.2
+					s0-6.5,0-11.6c0-5.1,0-12.1,0-21.1h16.9v39.4c0.1,9.6,1.7,16.2,4.8,19.9c3.1,3.7,7.7,5.4,13.8,5.2c3,0.1,5.8-0.2,8.4-1
+					c2.5-0.8,4.7-2.1,6.5-4.1c1.8-2,3.2-4.6,4.2-7.9c1-3.3,1.5-7.4,1.5-12.3v-39.2h16.9v34.6c0,11.9-3.2,21.3-9.5,28.2
+					c-6.3,6.9-15.4,10.3-27.2,10.3c-6.7,0-12.3-0.9-16.9-2.7c-4.6-1.8-8.3-4.4-11.2-7.8c-3.1,3.6-7,6.4-11.7,8.3
+					c-4.7,1.9-10.1,2.9-16.2,2.9c-6.6,0-12.1-0.9-16.7-2.7c-4.6-1.8-8.3-4.4-11.2-7.7c-2.9-3.3-5-7.4-6.3-12.1
+					c-1.3-4.7-2-10.1-2-16.1v-35.2h16.9v39.7C459.4,661.5,459.8,665.6,460.6,668.9z"/>
+			</g>
+			<text transform="matrix(0.939 0 0 1 368.2723 605.3397)" class="st3 st4 st5">media</text>
+		</g>
+	</g>
+</g>
+<g id="hilfslinien" class="st6">
+</g>
+</svg>
```

Comment:
Please optimize the SVG as described in the README.

Reply:
Have you run this through [SVGOMG](https://jakearchibald.github.io/svgomg/) yet?

Reply:
Yes.

Reply:
Hm, weird, it still has groups and is not minified. Oh well ;)

Reply:
I removed the file completely and re-added it, now I seems like it's the right svg-file. When is this pull-request going to be merged?